### PR TITLE
docs: subtle wording changes for DXA3

### PR DESCRIPTION
**Pull request by kimberlyxdavis**
## PR Checklist

- [ ] Leverages formatting as provided by https://squidfunk.github.io/mkdocs-material/reference/abbreviations/ 
- [ ] Have viewed and verified edits in rendered form ensuring proper formatting

## What are you changing?

 .\docs\Docker\dockerLocal.md
.\docs\General\buildRPC.md  
 .\docs\General\createCIRAConfig.md
.\docs\Microservices\RPC\buildRPC_Manual.md
 .\docs\Microservices\overview.md
.\docs\Topics\mqtt.md
 .\docs\Tutorials\apiTutorial.md
.\mkdocs.yml

### CODE REVIEW

File: docs/Topics/mqtt.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Message Queuing Telemetry Transport (MQTT) Eventing Support
 
-Open AMT Cloud Toolkit supports Eventing, customizable event monitoring, using Message Queuing Telemetry Transport (MQTT), an IoT publish-and-subscribe network protocol. With MQTT Eventing, administrators subscribe and customize network event monitoring. This eliminates the need to query or poll MPS to determine network events, such as a device's activation or deactivation. Administrators can subscribe to events and respond proactively. 
+Open AMT Cloud Toolkit supports Eventing, customizable event monitoring, using Message Queuing Telemetry Transport (MQTT), an IoT publish-and-subscribe network protocol. With MQTT Eventing, administrators subscribe and customize network event monitoring of server events. This eliminates the need to query or poll MPS to determine network events, such as a device's activation or deactivation. Administrators can subscribe to events and respond proactively.
```
Comment (matt-primrose) [inline docs/Topics/mqtt.md:5]: I'm not sure what customize means here.  Users can't customize the events they receive.  They can subscript to specific topics (categories of events).

Comment (bwendlandt-intel) [reply docs/Topics/mqtt.md:5]: So something like, "With MQTT Eventing, administrators can subscribe to specific topics (categories of events) for server event monitoring."?  I think the meaning of customize here == pick-and-choose which events you want.

Comment (kimberlyxdavis) [reply docs/Topics/mqtt.md:5]: Yeah, I was thinking the customization was that they could make a decision what to subscribe to-- the categories. Is that wrong? The reason I use the word "customize" is that it generally speaks to having agency over something, and that tends to make people more satisfied than if they feel blocked by something.

Comment (kimberlyxdavis) [reply docs/Topics/mqtt.md]: @bwendlandt-intel @matt-primrose I think the latest changes address your concerns.

**Review by bwendlandt-intel — APPROVED** [review]

**Pull request merged**